docs/reference/security/fips-java17.asciidoc

```diff
@@ -1,10 +1,7 @@
-{es} {version} requires Java 17 or later.  
-There is not yet a FIPS-certified security module for Java 17 
-that you can use when running {es} {version} in FIPS 140-2 mode.
-If you run in FIPS 140-2 mode, you will either need to request
-an exception from your security organization to upgrade to {es} {version}, 
-or remain on {es} 7.x until Java 17 is certified. 
-ifeval::["{release-state}"=="released"]
+{es} 8.0+ requires Java 17 or later. {es} 8.12+ has been tested with https://www.bouncycastle.org/java.html[Bouncy Castle]'s Java 17
```

Comment:
In other sections we mention `8.13` instead, should we make this consistent?

Reply:
+1 makes sense to only list one version https://github.com/elastic/elasticsearch/pull/105041/commits/bd9a45fdff478fdf64ba310e72aa391e084f62c2